atd-vzv/src/views/map/MapPolygonFilter.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useRef } from "react";
+import { Editor, EditorModes } from "react-map-gl-draw";
+import { getFeatureStyle, getEditHandleStyle } from "./map-style";
+import { stringify as stringifyGeoJSON } from "wellknown";
+import styled from "styled-components";
+
+const MapPolygonFilter = ({ setMapPolygon }) => {
+  const _editorRef = useRef();
+  const [mode, setMode] = useState(EditorModes.READ_ONLY);
+  const [features, setFeatures] = useState([]);
+
+  const StyledDrawTools = styled.div`
+    .disabled {
+      filter: opacity(0.3) drop-shadow(0 0 0 #fff);
+      cursor: default;
+    }
+  `;
+
+  const _onUpdate = ({ editType }) => {
+    const newFeatures = _editorRef.current.getFeatures();
+    setFeatures(newFeatures);
+
+    if (editType === "addFeature") {
+      // Convert polygon to Well-Known Text format
+      const feature = stringifyGeoJSON(newFeatures[0]);
+      setMapPolygon(feature);
+      setMode(EditorModes.READ_ONLY);
+    }
+  };
+
+  const _onDelete = () => {
+    // If a polygon exists in Editor features, delete it
+    if (features.length > 0) {
+      _editorRef.current.deleteFeatures(0);
+    }
+    // Remove filter from Socrata query and reset state
+    setMapPolygon(null);
+    setFeatures([]);
+  };
+
+  const _renderDrawTools = () => {
+    // Disable draw button click handler and grey out button if a polygon is drawn
+    const isDisabled = features.length > 0;
+    return (
+      <StyledDrawTools>
+        <div className="mapboxgl-ctrl-top-right">
+          <div className="mapboxgl-ctrl-group mapboxgl-ctrl">
+            <button
```

Comment:
I wonder how much this matters, but I think it may be a good practice for a11y... 

The same way we give the button a `disabled` class for CSS updates, we can give the `<button>` element a `disabled` attribute for better semantic markup:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled

Reply:
This is definitely the right way to go! Added. ✅